src/EditorFeatures/CSharpTest/Completion/CompletionProviders/AwaitCompletionProviderTests.cs

```diff
@@ -24,19 +24,40 @@ public class AwaitCompletionProviderTests : AbstractCSharpCompletionProviderTest
     {
         internal override Type GetCompletionProviderType() => typeof(AwaitCompletionProvider);
 
+        private const string CompletionDisplayTextAwait = "await";
+        private const string CompletionDisplayTextAwaitAndConfigureAwait = "awaitf";
+
         private async Task VerifyAbsenceAsync(string code)
         {
-            await VerifyItemIsAbsentAsync(code, "await");
+            await VerifyItemIsAbsentAsync(code, CompletionDisplayTextAwait);
+            await VerifyItemIsAbsentAsync(code, CompletionDisplayTextAwaitAndConfigureAwait);
         }
 
         private async Task VerifyAbsenceAsync(string code, LanguageVersion languageVersion)
         {
-            await VerifyItemIsAbsentAsync(GetMarkup(code, languageVersion), "await");
+            await VerifyItemIsAbsentAsync(GetMarkup(code, languageVersion), CompletionDisplayTextAwait);
+            await VerifyItemIsAbsentAsync(GetMarkup(code, languageVersion), CompletionDisplayTextAwaitAndConfigureAwait);
         }
 
-        private async Task VerifyKeywordAsync(string code, LanguageVersion languageVersion, string? inlineDescription = null)
+        private async Task VerifyKeywordAsync(string code, LanguageVersion languageVersion, string? inlineDescription = null, bool dotAwait = false, bool dotAwaitf = false)
```

Comment:
when would dotAwait be true, but dotAwaitf be false?  in other words, why is this not just a single option that both items are available?

Reply:
see https://github.com/dotnet/roslyn/pull/55817#discussion_r704280089